bluebrain/repo-bluebrain/packages/brion/package.py

```diff
@@ -31,11 +31,14 @@ class Brion(CMakePackage):
     depends_on("py-numpy", type=("build", "run", "test"), when="+python")
 
     depends_on(
-        "boost +date_time+filesystem+iostreams+program_options+regex+shared+system+test",
+        "boost@:1.73 +date_time+filesystem+iostreams+program_options+regex+shared+system+test",
         when="~python",
     )
     depends_on(
-        "boost +date_time+filesystem+iostreams+program_options+regex+shared+system+test+python",
+        (
+            "boost@:1.73 +date_time+filesystem+iostreams+program_options+regex+shared"
```

Comment:
why are these pinned?

Reply:
My best guess is there was an issue with a newer version.